### PR TITLE
Fix/workflow execution exception

### PR DESCRIPTION
**Pull request by muhammad-ali-e**
## What

- Fix to handle unhandled exceptions in celery hence API Deployment

## Why

-

## How

- Raised exception from celery task 

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- Need to test Pipiline and API executions

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: backend/workflow_manager/workflow/workflow_helper.py [hunk 1]
```diff
@@ -531,13 +534,15 @@ def execute_bin(
...
-                    error=str(e),
+            except Exception as error:
+                error_message = traceback.format_exc()
+                logger.error(
+                    f"Error executing execution {workflow_execution}: {error_message}"
+                )
+                WorkflowExecutionServiceHelper.update_execution_err(
+                    execution_id, str(error)
                 )
+                raise
```
Comment (johnyrahul) [inline backend/workflow_manager/workflow/workflow_helper.py:545]: @muhammad-ali-e  
```suggestion
                raise error
```

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM, apart from API deployments and `Sync Now` in pipelines will this code be called anywhere else?

File: backend/workflow_manager/workflow/workflow_helper.py (hunk 1 above)
Comment (muhammadalie) [reply backend/workflow_manager/workflow/workflow_helper.py:545]: Both are okay. Raise will raise the same exception it handled

**Comment by muhammadalie** [comment]
> LGTM, apart from API deployments and `Sync Now` in pipelines will this code be called anywhere else?

Every pipeline executions

**Review by johnyrahul — APPROVED** [review]

**Pull request merged**